### PR TITLE
Support for 2.3 to display script tag in frontend

### PR DESCRIPTION
**Pull request by nikolalardev**

### CODE REVIEW

File: view/frontend/templates/adsccout-js.phtml [hunk 1]
```diff
@@ -9,9 +9,17 @@ $viewModel = $block->getAdscoutViewModel();
 
 <?php if ($viewModel->isEnabled() == 1): ?>
     <?php if ($viewModel->getIntegrationType() == 1): ?>
-        <script async src="https://adscout.<?= $viewModel->getDomain()?>/script/<?= $viewModel->getApiCode()?>"></script>
+        <script async
+                src="https://adscout.<?= $viewModel->getDomain() ?>/script/<?= $viewModel->getApiCode() ?>"></script>
     <?php else: ?>
-        <?php //TODO compatible for under 2.4 ?>
-        <?= /* @noEscape */ $secureRenderer->renderTag('script', ['async' => '', 'src' => 'https://scoutefy.com/script/'. $viewModel->getApiCode()], null); ?>
+        <?php if ($viewModel->isMagentoVersionBelow24()): ?>
```
Comment (adscout-io) [inline view/frontend/templates/adsccout-js.phtml:15]: This is not going to work fo Integration Type with DNS, should we put it above as well?

File: view/frontend/templates/adsccout-js.phtml [hunk 2]
```diff
@@ -9,9 +9,17 @@ $viewModel = $block->getAdscoutViewModel();
...
+            <script async=""
+                    src="https&#x3A;&#x2F;&#x2F;scoutefy.com&#x2F;script&#x2F;<?= $viewModel->getApiCode() ?>"></script>
```
Comment (adscout-io) [inline view/frontend/templates/adsccout-js.phtml]: There are strange characters here, is this OK?

**Review by adscout-io — COMMENTED** [review]
I have added two comments in phtml file

File: view/frontend/templates/adsccout-js.phtml (hunk 2 above)
Comment (nikolalardev) [reply view/frontend/templates/adsccout-js.phtml]: It was hex code characters, but I changed to normal.

File: view/frontend/templates/adsccout-js.phtml (hunk 1 above)
Comment (nikolalardev) [reply view/frontend/templates/adsccout-js.phtml:15]: Integration type with DNS record logic is on line 12-13,
on lint 15-18 is Integration type without DNS but for Magento under 2.4 and from line 18-23 is for Magento 2.4 or higher, where the new functionality renderTag is introduced.
\Magento\Framework\View\Helper\SecureHtmlRenderer

**Pull request merged**